### ISSUE TITLE
AttributeError: 'UserConnection' object has no attribute 'nick'

### ISSUE THREAD

**Issue by Akuli** [ThePhilgrim/MantaTail#77]
Start MantaTail, connect to it with netcat, and then immediately disconnect without send anything. You will get error

```
Exception in thread Thread-71:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/home/akuli/MantaTail/mantatail.py", line 253, in send_queue_thread
    self.queue_quit_message_for_other_users()
  File "/home/akuli/MantaTail/mantatail.py", line 289, in queue_quit_message_for_other_users
    if channel.is_operator(self):
  File "/home/akuli/MantaTail/mantatail.py", line 376, in is_operator
    return user.nick.lower() in self.operators
AttributeError: 'UserConnection' object has no attribute 'nick'
```

**Comment by Akuli** [comment]
Actually, reproducing the bug needs a different user to join some channel:

1. In one terminal, send `NICK x`, `USER x`, `JOIN #foo`
2. In a second terminal, close netcat without sending anything.

**Comment by Akuli** [comment]
Already fixed in #79, but it would be good to add a test.
